app/controllers/recipie_controller.rb

```diff
@@ -1,9 +1,31 @@
 class RecipieController < ApplicationController
   before_action :authenticate_user!
   def show
-    @recipie = Recipie.find(params[:id])
+    @recipie = Recipie.find(params[:id])  
     @steps = @recipie.steps
+    @comments = []
+    @steps.each do |step|
+      @comments << step.comments.order("created_at")
```

Comment:
Are you sure you are using the right order by clause?

Reply:
hmm ! Yes , before i was hardcoding the "comment_num" in the database and sorting based on that , but in one of the PR raised by gagan it said this is not the right way . So then i used this and removed comment_num in DB .